src/main.c

```diff
@@ -227,6 +227,11 @@ int main(int argc, char **argv) {
 
   const char *init_lite_code = \
     "local core\n"
+    "local os_exit = os.exit\n"
+    "os.exit = function(code, close)\n"
+    "  if type(close) ~= 'boolean' then close = true end\n"
+    "  os_exit(code, close)\n"
+    "end\n"
```

Comment:
Would
```suggestion
    "local os_exit = os.exit\n"
    "os.exit = function(code, close)\n"
    "  os_exit(code, close == nil and true)\n"
    "end\n"
```
work?

Reply:
From the lua interactive shell:

```
> close = true
> close == nil and true
false
```

Reply:
Ah yeah, try `close == nil and true or close`.

Reply:
yeah that one works, want to change your suggestion then to commit it directly?

Reply:
Feel free to manually commit it.